.github/workflows/release.yaml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install Dependencies
+        run: yarn install
+
+      - name: Build
+        run: yarn build
+
+      - name: Publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
+        run: yarn publish
+
+      - name: Push version tag
+        run: yarn release:tag
```

Comment:
@sudoryan Just to clarify - the version will be taken from package.json script - correct? So we would need to update version manually before making github release.

Reply:
Yes, we need to update version in package.json first

Reply:
It would be better to ask what kind of release user wants to do, like we do when we deploy production with a list of questions:

https://monosnap.com/direct/GlPzmEdQXhsqRi6W5gHfbpNHStOzJC